VideoWeb/VideoWeb/Controllers/UserDataController.cs

```diff
@@ -55,6 +56,10 @@ public async Task<ActionResult<IList<CourtRoomsAccountResponse>>> GetCourtRoomsA
                 var allocatedHearings =
                     await _bookingApiClient.GetAllocationsForHearingsAsync(conferences.Select(e => e.HearingRefId));
                 var conferenceForVhOfficerResponseMapper = _mapperFactory.Get<ConferenceForAdminResponse, AllocatedCsoResponse, ConferenceForVhOfficerResponse>();
+                
+                if(query.IncludeUnallocated)
+                    conferences = conferences.Where(c => c.CaseType != "Generic").ToList();
```

Comment:
you'll need to apply this change to the [GetConferencesForVhOfficerAsync endpoint](https://github.com/hmcts/vh-video-web/blob/master/VideoWeb/VideoWeb/Controllers/ConferencesController.cs#L154) too for them to be filtered out of the command centre. It might be worth moving it to `ConferenceForVhOfficerResponseExtensions` to avoid duplication

Reply:
Sure thing, not sure why I didn't do it that way in the first place